integration-tests/main.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/docker/go-connections/nat"
+)
+
+var (
+	cli        *client.Client
+	containerID string
+)
+
+const (
+	PROJECT_DIR          = "../"
+	DOCKERFILE          = "../Dockerfile"
+	IMAGE_NAME          = "autodba:latest"
+	CONTAINER_NAME      = "autodba_test"
+	PROMETHEUS_PORT     = "9090"
+	BFF_PORT            = "4000"
+	DB_CONN_STRING      = "postgres://postgres:lTEP7OzeXQr77Ldu@mohammad-dashti-rds-1.cvirkksghnig.us-west-2.rds.amazonaws.com:5432/postgres?sslmode=require"
+	AWS_RDS_INSTANCE    = "mohammad-dashti-rds-1"
+	DEFAULT_METRIC_PERIOD = "5"
+	WARM_UP_TIME        = "60"
+	CONFIG_FILE         = "/home/steams/Development/autodba/autodba-collector.conf"
+	BACKUP_DIR          = "/home/autodba/ext-backups"
+	BACKUP_FILE         = ""
+    // -e AWS_ACCESS_KEY_ID="" \
+    // -e AWS_SECRET_ACCESS_KEY="" \
+    // -e AWS_REGION="" \
+    // -e DISABLE_DATA_COLLECTION="false" \
```

Comment:
I know it's WIP, but this should be a config struct, which we can pass multiple of them to the test and test different targets.